IDEAS/Examples/Tutorial/DetailedHouse/package.mo

```diff
@@ -0,0 +1,107 @@
+i.e., no errors should be produced and simulation results may be compared.
+</p>
+<p>
+The model has been created in the following stages:
+</p>
+<ol>
+<li>
+<a href=\"modelica://IDEAS.Examples.Tutorial.DetailedHouse.DetailedHouse1\">
+IDEAS.Examples.Tutorial.DetailedHouse.DetailedHouse1</a>
+represents a basic building model that includes a single zone, four walls, a window, a floor, and a ceiling
+and serves as a starting model to implement the entire <code>DetailedHouse</code> model.
+</li>
+<li>
+<a href=\"modelica://IDEAS.Examples.Tutorial.DetailedHouse.DetailedHouse2\">
+IDEAS.Examples.Tutorial.DetailedHouse.DetailedHouse2</a>
+implements solar shading extensions by adding a screen to the window model.
+</li>
+<li>
+<a href=\"modelica://IDEAS.Examples.Tutorial.DetailedHouse.DetailedHouse2\">
+IDEAS.Examples.Tutorial.DetailedHouse.DetailedHouse3</a>
+adds continuous occupancy of 1 person and LED lighting for the zone.
+</li>
+<li>
+<a href=\"modelica://IDEAS.Examples.Tutorial.DetailedHouse.DetailedHouse4\">
+IDEAS.Examples.Tutorial.DetailedHouse.DetailedHouse4</a>
+adds an occupancy schedule that assigns two occupants during office hours and 
+zero otherwise. This schedule will be created using extensions of the IDEAS 
+templates and replaceable models.
+</li>
+<li>
+<a href=\"modelica://IDEAS.Examples.Tutorial.DetailedHouse.DetailedHouse5\">
+IDEAS.Examples.Tutorial.DetailedHouse.DetailedHouse5</a>
+This example teaches the use of the <code>RectangularZoneTemplate</code> .
+</li>
+<li>
+<a href=\"modelica://IDEAS.Examples.Tutorial.DetailedHouse.DetailedHouse6\">
+IDEAS.Examples.Tutorial.DetailedHouse.DetailedHouse6</a>
+adds a HVAC system. The system consists of a water-water heat pump, radiators, a storage tank, circulation
+pumps and a low temperature heat source for the heat pump.
+</li>
+<li>
+<a href=\"modelica://IDEAS.Examples.Tutorial.DetailedHouse.DetailedHouse7\">
+IDEAS.Examples.Tutorial.DetailedHouse.DetailedHouse7</a>
+adds a controller that disables the heat pump when the supply water
+temperature exceeds 45◦C.
+</li>
+<li>
+<a href=\"modelica://IDEAS.Examples.Tutorial.DetailedHouse.DetailedHouse8\">
+IDEAS.Examples.Tutorial.DetailedHouse.DetailedHouse8</a>
+adds workflow automation, generating a CSV file at a userdefined location that contains data 
+for each of the inputs of the block .
+</li>
+<li>
+<a href=\"modelica://IDEAS.Examples.Tutorial.DetailedHouse.DetailedHouse9\">
+IDEAS.Examples.Tutorial.DetailedHouse.DetailedHouse9</a>
+adds a CO2-controlled ventilaation system. The occupancy model from <a href=\"modelica://IDEAS.Examples.Tutorial.DetailedHouse.DetailedHouse4\">
+IDEAS.Examples.Tutorial.DetailedHouse.DetailedHouse4</a> is added to one zone and a 
+fixed occupancy of 1 person to the other zone. The ventilation system
+consists of two fans, two supply and two return air VAVs (Variable Air Volume), a heat recovery unit and an
+outdoor air source. The control consists of PI controllers with a set point of 1000 ppm.
```

Comment:
setpoint (maybe also apply this change to the exercise sheet of the crash course)